package.json

```diff
@@ -91,6 +91,19 @@
       "coverageReporters": [
         "json"
       ],
+      "collectCoverageFrom": [
+        "packages/src/**/*{.ts,tsx}",
```

Comment:
should this be `packages/src/**/*.{ts,tsx}`? (period outside the brackets)

might be worth looking at the generated `jest.config.js`, locally mine already looks like this so some of this may be duplicated. might be worth ignoring `test/**/*.{ts,tsx}` also?

```
  ...
  "collectCoverageFrom": [
    "**/src/**/*.{ts,tsx,js,jsx}",
    "**/test/**/*.{ts,tsx,js,jsx}"
  ],
  "coverageDirectory": "./coverage",
  "coveragePathIgnorePatterns": [
    "coverage/",
    "node_modules/",
    "public/",
    "esm/",
    "lib/",
    "tmp/",
    "dist/"
  ],
  ...
```